### ISSUE TITLE
Add small tag to list of allowed HTML tags

### ISSUE THREAD

**Issue by mxamber** [glitch-soc/mastodon#2131]
### Pitch

The `sub` and `sup` tags are supported, but `small` isn't. I propose adding `small` to the allowlist.

### Motivation

I can't think of any reason to prohibit `small` that wouldn't also apply to `sub` and `sup`?

**Comment by Plastikmensch** [comment]
Looks to me this isn't intentional.
Adding the small tag requires styling though and it makes sense to wait until #2122 is merged as it refactors the styling for rich text.

**Comment by fasiha** [comment]
Is there a place to have a broader conversation about tags that are currently blocked? I'd love to have `<ruby>` and `<rt>` tags for <ruby>東<rt>East</rt></ruby> <ruby>亜<rt>Asian</rt></ruby> <ruby>語<rt>languages</rt></ruby>. Potentially related: #1349?

**Comment by Plastikmensch** [comment]
> Is there a place to have a broader conversation about tags that are currently blocked? I'd love to have `<ruby>` and `<rt>` tags (see [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ruby)) for 東East 亜Asian 語languages. Potentially related: #1349?
> 
> > Rough thoughts. `<ruby>` and `<rt>` I _think_ work seamlessly in (most?) browsers without needing styling, but since `<small>` does too, I'm guessing all allowed tags need explicit styling in Glitch-soc/Mastodon?

Not all tags require styling, but some need some adjustment to display correctly.
This too, because the font size is too small by default:
![Screenshot_2023-03-12_23-13-04](https://user-images.githubusercontent.com/29483052/224576886-4810df08-b49f-4643-841a-63c7d4084ed9.png)

I'm not opposed to including more html tags and this issue seems like a good place to have a broader discussion about which tags to support going forward.

**Comment by fasiha** [comment]
Thanks @Plastikmensch! So just to confirm, the `rt` text is intended to be smaller than the main body text, including the non-`rt` body of `ruby`, just like how GitHub renders it above—forgive me if I’m mansplaining something you already know! I say all this because your screenshot looks totally acceptable to my eyes: 明日 seems to be the same size as other characters in the toot (“Ruby annotation”), and “Ashita” is small. For reference, here’s GitHub rendering it: <ruby>明日<rt>Ashita</rt></ruby>.

**Comment by Plastikmensch** [comment]
> Thanks @Plastikmensch! So just to confirm, the `rt` text is intended to be smaller than the main body text, including the non-`rt` body of `ruby`, just like how GitHub renders it above—forgive me if I’m mansplaining something you already know! I say all this because your screenshot looks totally acceptable to my eyes: 明日 seems to be the same size as other characters in the toot (“Ruby annotation”), and “Ashita” is small. For reference, here’s GitHub rendering it: 明日Ashita.

Yes, I'm just very much opposed to any font size < 10px because I find it very hard to read.
I would prefer font-size 12px:
![Screenshot_2023-03-12_23-45-51](https://user-images.githubusercontent.com/29483052/224578499-a5bb21ae-94d1-4ebf-95cb-991b05373e0a.png)
